dv/uvm/core_ibex/riscv_dv_extension/ibex_log_to_trace_csv.py

```diff
@@ -205,7 +205,7 @@ def check_ibex_uvm_log(uvm_log):
         test_result_seen = False
 
         for line in log:
-            if ('UVM_ERROR' in line or 'UVM_FATAL' in line) \
+            if ('UVM_ERROR' in line or 'UVM_FATAL' in line or 'Error' in line) \
```

Comment:
Can you add a note to the commit message that explains what this is doing? I don't think it's covered by what's there at the moment.